### ISSUE TITLE
How many GPUs are used when  you train the models here?   I have 4 gpus.  How can I set the learning rate here ?

### ISSUE THREAD

**Issue by shuyuandeqipa** [swathikirans/GSM#24]

**Comment by swathikirans** [comment]
I used 2 GPUs with gradient aggregation to have an effective batch size of 32. You can use all 4 GPUs without gradient aggregation to run with the same setup used in the paper. Change "iter_size" to 1 to have this.
